rustls-libssl/tests/runner.rs

```diff
@@ -329,6 +329,80 @@ fn server() {
     assert_eq!(openssl_output, rustls_output);
 }
 
+fn server_with_key_algorithm(key_type: &str, sig_algs: &str, version_flag: &str) {
+    fn connect(key_type: &str, sig_algs: &str, version_flag: &str) {
+        Command::new("openssl")
+            .env("LD_LIBRARY_PATH", "")
+            .args([
+                "s_client",
+                "-connect",
+                "localhost:5556",
```

Comment:
WDYT about pulling out a `port` binding to avoid repeating the constant ~3 times.

I think this is a pre-existing pattern I missed giving feedback on earlier, so it also applies to other parts of `runner.rs`. With that in mind, ignoring the feedback for consistency and considering this for cleanup later seems fine.

Reply:
Yeah the other thing about ports here is they need to be unique across all parallelised tests. That's why the tests added in this PR run serially -- just to avoid that mess. Will keep that in mind as an improvement here (maybe we could have a `fn allocate_port() -> u16` that ensures they are unique without being excessively hardcoded.)